cluster/manifests/logging-agent/daemonset.yaml

```diff
@@ -244,4 +250,7 @@ spec:
       - name: fluentd-posfiles
         hostPath:
           path: /var/lib/fluentd
+
+      - name: scalyr-agent-logs
```

Comment:
When this directory is [re]created? On pod restart? Or on container restart?

Reply:
https://kubernetes.io/docs/concepts/storage/volumes/#emptydir